### DISCUSSION TITLE
Doubt in Stable Coin Project (Lesson 12), DSCEngine.sol

### DISCUSSION THREAD

**Discussion by karthiksathishjeemain** [Cyfrin/foundry-full-course-cu#1081] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/115690993/a6845c3a-0aac-4a05-9933-60e790e187d7)
In here, what I feel is, by  calling  the "revertIfHealthFactorIsBroken(msg.sender); " function,  healthfactor becomes undefined as it's denominator has the term totalDSCminted which eventually becomes zero if all the stablecoins are burned 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/115690993/639c9d7c-eb55-48a9-b279-266e7fd43e16)
I am feeling this because, we got the totalDSCminted by the s_DSCMinted mapping which we are updating when we call  _burnDSC();
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/115690993/6745a9ae-105a-4cd8-ac04-96291f139eca)

**Comment by Niferu** [comment]
Hi @karthiksathishjeemain 

But what is your question actually?  I can confirm this will not revert as we can break healthFactor by increasing coin minted not decreasing it's amount.
